warehouse/templates/packaging/detail.html

```diff
@@ -244,7 +244,8 @@ <h2 class="no-top-padding">Reverse Dependencies (TODO)</h2>
             <div class="release{{ ' release--latest' if loop.first else ' release--oldest' if loop.last }}{{ ' release--current' if is_current }}">
               <div class="release__meta">
                 {% if is_current %}
-                <p class="release__version">{{ hrelease.version }}</p>
+                <p class="release__version"><a href="{{ request.route_path('packaging.project', name=release.project.name) }}">
+                {{ hrelease.version }}</a></p>
```

Comment:
I think the creator of #1504 was hoping that this link would have the version number in it as well, so it would be easy to get a hard link to any possible version from the "Release History" tab.